straxen/analyses/event_display.py

```diff
@@ -93,16 +114,18 @@ def event_display(context,
     if records_matrix not in ('raw', True, False):
         raise ValueError('Choose either "raw", True or False for records_matrix')
     if ((records_matrix == 'raw' and not context.is_stored(run_id, 'raw_records')) or
-            (records_matrix and not context.is_stored(run_id, 'records'))):
+        (isinstance(records_matrix, bool) and records_matrix and not context.is_stored(run_id, 'records'))):   # noqa
         print("(raw)records not stored! Not showing records_matrix")
         records_matrix = False
     if not context.is_stored(run_id, 'peaklets'):
         raise strax.DataNotAvailable(f'peaklets not available for {run_id}')
 
-    fig = plt.figure(figsize=(25, 7 * (2 + int(records_matrix))),
+    # String convert for int(_rr_bool)
+    _rr_bool = bool(records_matrix)

```

Comment:
You can also convert it in one go into an `int` is not it? I think you are not using `_rr_bool`  anymore.

Reply:
Ah good point. Thanks Daniel!